src/Auth0.php

```diff
@@ -803,4 +803,25 @@ public function setDebugger(\Closure $debugger)
     {
         $this->debugger = $debugger;
     }
+    
+    /**
+     * Gets the store interface
+     *
+     * @return StoreInterface
+     */
+    public function getStore(): StoreInterface 
```

Comment:
The master branch here targets 5.5, so no return type hinting. We are working on a major in the `7.0.0-dev` branch but if you want these methods out before then, they'll have to be 5.5 compatible.

Reply:
removed the return type hinting